vars/piperExecuteBin.groovy

```diff
@@ -54,7 +54,7 @@ void call(Map parameters = [:], String stepName, String metadataFile, List crede
             config += ["ansHookServiceKeyCredentialsId": ansHookServiceKeyCredentialsId]
 
             // prepare stashes
-            // first eliminate empty stashes
+            // first eliminate non existing stashes
```

Comment:
by the way:

- unstashing everything just to remove non existing stashes from `stashContent' looks like overkill wrt resource consumtion (disk-space, time).
- what we unstash here might cause side effects. Would be better to unstash into some temp dir which gets deleted afterwards.